docs/source/conf.py

```diff
@@ -23,9 +23,9 @@
 author = "Ciela Institute"
 
 # The short X.Y version
-version = "0.5"
+version = __version__
```

Comment:
This will give the full version and I think it's only supposed to be `MAJOR.MINOR` so maybe we need something like: 
```suggestion
version = __version__[:__version__.find(".", __version__.find(".")+1)]
```

Reply:
I think I'll leave this for now. As I mentioned, we're currently working on an update to the docs stuff 😄